Assets/Scripts/Utilities/RenderSettingsManager.cs

```diff
@@ -46,6 +44,15 @@ void Awake()
 
 	void Update()
 	{
+		dayCycleTimer += Time.deltaTime;
+
+		timeOfDay = (-Mathf.Cos(dayCycleTimer/dayCycleTime * Mathf.Rad2Deg) * 0.5f) + 0.5f;
```

Comment:
what's the Rad2Deg doing in there? Mathf.Cos takes in radians. http://docs.unity3d.com/ScriptReference/Mathf.Cos.html